packages/cli/src/commands/database/config.ts

```diff
@@ -41,6 +42,8 @@ const validRotateKeys = Object.freeze([
   LogtoOidcConfigKey.CookieKeys,
 ] as const);
 
+const validPrivateKeyTypes = Object.freeze(['rsa', 'ec'] as const);
```

Comment:
align with the generateOidcPrivateKey argument type?  Keep a SSOT

Reply:
updated